<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stock</class>
 <widget class="QDialog" name="stock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1788</width>
    <height>1048</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 75 10pt &quot;MV Boli&quot;;
}
#stock
{

background-color: QLinearGradient(x1: 0,y1: 0,x2: 0,y2: 1,stop: 0 #e66465, stop:1 #9198e5);
border-radius:15px;
}
QLabel
{
	color: rgb(0, 0, 0);
}
QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
QPushButton
{
border-radius:15px;
}
QPushButton:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}
Qlabel{
height:50px;
width:40px;
}

QToolButton
{
background: transparent;
border-radius:60px;

}</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>1121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0)
</string>
   </property>
   <widget class="QPushButton" name="pushButton_ajout_image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ajouter image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_notification">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Notification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

color: rgb(0, 0, 0);

font: 14pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion des stocks</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>1651</width>
     <height>901</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
font: 75 10pt &quot;MV Boli&quot;;
}
#widget
{
border-radius:15px;
}
QLabel
{
	color: rgb(0, 0, 0);
}
QPushButton
{
	
	background-color: rgb(214, 143, 0);
	border-radius:60px;
}
QPushButton
{
border-radius:15px;
}
QPushButton:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}
Qlabel{
height:50px;
width:40px;
}

QToolButton
{
background: transparent;
border-radius:60px;

}</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>1171</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>511</width>
      <height>821</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <widget class="QLineEdit" name="lineEdit_TYPE">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_QUANTITE">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DATE_ARRIVE">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_annuler">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>490</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_annuler
{
border-radius:15px;
}
#pushButton_annuler:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date d'arrive</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>490</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_ajouter
{
border-radius:15px;
}
#pushButton_ajouter:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>410</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Validite</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_VALIDITE_DU_AIDE">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>430</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>540</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_supprimer
{
border-radius:15px;
}
#pushButton_supprimer:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>540</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_modifier
{
border-radius:15px;
}
#pushButton_modifier:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Num aide</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NUM_AIDE">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pdf">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>590</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_pdf
{
border-radius:15px;
}
#pushButton_pdf:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
     </property>
     <property name="text">
      <string>pdf</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rechercher">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>640</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_rechercher
{
border-radius:15px;
}
#pushButton_rechercher:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_statistique">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>640</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_statistique
{
border-radius:15px;
}
#pushButton_statistique:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
     </property>
     <property name="text">
      <string>Statistique</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trier">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>590</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_trier
{
border-radius:15px;
}
#pushButton_trier:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_afficher">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>710</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
#pushButton_afficher
{
border-radius:15px;
}
#pushButton_afficher:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}</string>
    </property>
    <property name="text">
     <string>Afficher la liste</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>140</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Chercher par type</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Déconnecter</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/logout.png</normaloff>../../Desktop/logout.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
